ci/init.js

```diff
@@ -11,9 +13,20 @@ const options = {
   }
 }
 
-if (process.env.DD_CIVISIBILITY_AGENTLESS_ENABLED && (process.env.DATADOG_API_KEY || process.env.DD_API_KEY)) {
-  options.experimental = {
-    exporter: 'datadog'
+let shouldInit = true
+
+if (process.env.DD_CIVISIBILITY_AGENTLESS_ENABLED) {
+  if (process.env.DATADOG_API_KEY || process.env.DD_API_KEY) {
+    options.experimental = {
+      exporter: 'datadog'
+    }
+  } else {
+    console.error(`
```

Comment:
Since the tracer is loaded, but not initialized, is the logger available at this point? If so, it should be used here. If not, then nevermind.

Reply:
that's a good question. I'll try

Reply:
indeed, `log.error` won't work because it's not enabled (https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/log.js#L115). Using console.error is a bit aggressive but the way to get it is not straightforward. You have to use ci visibility's entrypoint and opt in to agentless via the flag. At that point I think it's fine to warn users that they've messed up their setup (by missing their API key)